src/otx/core/exporter/visual_prompting.py

```diff
@@ -49,13 +49,6 @@ def export(  # type: ignore[override]
         Returns:
             dict[str, Path]: paths to the exported models
         """
-        # NOTE: Rather than using OTXModel.forward_for_tracing()
```

Comment:
It's just a question. Why isn't `forward_for_tracing` used before?

Reply:
Before refactoring, image encoder's forward is used as is and decoder's forward was decided by a flag.
So, we didn't need to implement `forward_for_tracing`.
Through refactoring, I moved decoder's forward for export to `forward_for_tracing`, so we can use `forward_for_tracing` for decoder.